app/models.py

```diff
@@ -312,5 +312,15 @@ class HTML(db.Model):
 	value = db.Column(db.Text)
 
 class ZipSearch(db.Model):
-	zip_code = db.Column(db.String(5), primary_key = True)
+	id = db.Column(db.Integer, primary_key = True)
```

Comment:
Yes, I agree that it was a great idea prior to the addition of the date column; however, with the addition of the date column, there will be multiple instances of (e.g.) '19104' in the database, as each will have a different corresponding date. How does that sound?